lib/vault/rails.rb

```diff
@@ -69,10 +74,12 @@ def respond_to_missing?(m, include_private = false)
       #   the plaintext to encrypt
       # @param [Vault::Client] client
       #   the Vault client to use
+      # @param [Bool] convergent
+      #   should use convergent encryption?
       #
       # @return [String]
       #   the encrypted cipher text
-      def encrypt(path, key, plaintext, client = self.client)
+      def encrypt(path, key, plaintext, client = self.client, convergent = false)
```

Comment:
#78 changes the signature of encrypt and decrypt, such that we can refactor this PR to use keyword arguments for the `convergent` boolean.